### DISCUSSION TITLE
Release Schedule

### DISCUSSION THREAD

**Discussion by trevornagy** [fnc12/sqlite_orm#1242] [General]
The project I'm working on requires FTS which is only on the dev branch right now. Ideally, I'd like to use an official release over the dev branch. When can we expect to see a release cut? Would it be possible to make a new release soon?

**Comment by trueqbit** [comment]
I would like to see the [CTE](../pull/1124) functionality included in the next version as well.

**Comment by fnc12** [comment]
@trevornagy we don't have exact version. Usually it depends on our free time. The only thing which is being done in time is paid customers requests. If you want to become on of them write me at fnc12@me.com. Otherwise please stay subscribed - we'll try to make a release soon.
BTW why don't you use `dev` branch as is instead of `master` or tags?

**Reply by trevornagy** [reply]
> BTW why don't you use dev branch as is instead of master or tags?

I was hoping to use vcpkg rather than having to download the dev branch and include that folder in my project.

**Reply by fnc12** [reply]
In a perfect world in my imagination vcpkg has to be able to give an opportunity to specify branch if we want just like SPM (Swift Package Manager) does. We (sqlite_orm team) will try to publish the next release ASAP but also I would recommend you to ask Microsoft for specifying branch in vcpkg cause it looks like an artificial restriction

**Reply by trueqbit** [reply]
@fnc12 Although vcpkg is managed by Microsoft, it is still a community effort. Typically, the maintainers of the libraries, or someone who wishes to do so instead, create and manage a "port" in the vcpkg ecosystem. Since vcpkg consists of a curated set of libraries, this is not an automatic process, but has to be done manually. I've done this the last couple of times. In my opinion, for any packaging solution, a maintainer approved release is the best a package ecosystem can rely on.
@trevornagy However, it is possible to create your own port. I usually maintain some for existing ports, just to be able to customize things for my needs. As for sqlite_orm, I also maintain one so I can get the latest stuff as quickly as possible. It's currently on the [CTE branch](https://gitlab.com/FireDaemon/open-source/vcpkg-firedaemon-ports/-/tree/master/ports/sqlite-orm) in the forked sqlite_orm repository.

**Comment by juandent** [comment]
I was also interested in the dev branch to download it from vcpkgCouldn’t we do this?Vcpkg is an amazing toolMy recommendation is to publish the dev branch in vcpkgThink about it!!RegardsJuan dentSent from my iPhoneOn 20 Nov 2023, at 2:27 PM, Trevor Nagy ***@***.***> wrote:﻿

BTW why don't you use dev branch as is instead of master or tags?

I was hoping to use vcpkg rather than having to download the dev branch and include that folder in my project.

**Comment by jairoareyes** [comment]
Hi all,

I would like to know if the V1.9 is ready. I see a branch for the release, but it is not listed in the release section.  Thanks in advance.

**Reply by fnc12** [reply]
Hi. Several days left - it is in progress

**Comment by jairoareyes** [comment]
Is it possible to add an option to not include the test at compilation? I'm having an issue when compiling the tests. 

![image](https://github.com/user-attachments/assets/d2e2a4f1-825b-409f-81ee-b3fe2bef1acd)
```
/home/devices-libs/sqliteorm/sqlite_orm/tests/static_tests/is_column_with_insertable_primary_key.cpp:41:20: error: ‘insertable_index_sequence’ was not declared in this scope
     STATIC_REQUIRE(insertable_index_sequence<decltype(insertable)>::size() == 4);
                    ^
/home/devices-libs/sqliteorm/sqlite_orm/tests/static_tests/is_column_with_insertable_primary_key.cpp:41:46: error: expected primary-expression before ‘decltype’
     STATIC_REQUIRE(insertable_index_sequence<decltype(insertable)>::size() == 4);
                                              ^
/home/devices-libs/sqliteorm/sqlite_orm/tests/static_tests/is_column_with_insertable_primary_key.cpp:41:46: error: expected ‘,’ before ‘decltype’
/home/devices-libs/sqliteorm/sqlite_orm/tests/static_tests/is_column_with_insertable_primary_key.cpp:41:46: error: expected string-literal before ‘decltype’
/home/devices-libs/sqliteorm/sqlite_orm/tests/static_tests/is_column_with_insertable_primary_key.cpp:41:46: error: expected ‘)’ before ‘decltype’
/home/devices-libs/sqliteorm/sqlite_orm/tests/static_tests/is_column_with_insertable_primary_key.cpp:42:20: error: ‘noninsertable_index_sequence’ was not declared in this scope

```

**Comment by fnc12** [comment]
here you go https://github.com/fnc12/sqlite_orm/releases/tag/v1.9

**Comment by fnc12** [comment]
So in the end we were late with `v1.9` but better late then never. I think we can close this discussion. Anyone please feel free to reopen it if something new appears
